sqlx-core/src/mssql/types/int.rs

```diff
@@ -7,30 +7,6 @@ use crate::mssql::protocol::type_info::{DataType, TypeInfo};
 use crate::mssql::{Mssql, MssqlTypeInfo, MssqlValueRef};
 use crate::types::Type;
 
-impl Type<Mssql> for i8 {
-    fn type_info() -> MssqlTypeInfo {
-        MssqlTypeInfo(TypeInfo::new(DataType::IntN, 1))
-    }
-
-    fn compatible(ty: &MssqlTypeInfo) -> bool {
-        matches!(ty.0.ty, DataType::TinyInt | DataType::IntN) && ty.0.size == 1
-    }
-}
-
-impl Encode<'_, Mssql> for i8 {
-    fn encode_by_ref(&self, buf: &mut Vec<u8>) -> IsNull {
-        buf.extend(&self.to_le_bytes());
-
-        IsNull::No
-    }
-}
-
-impl Decode<'_, Mssql> for i8 {
-    fn decode(value: MssqlValueRef<'_>) -> Result<Self, BoxDynError> {
-        Ok(value.as_bytes()?[0] as i8)
-    }
-}
-
```

Comment:
Removing these impls is a breaking change.

Reply:
Yeah, did not think about it.
Solved